tests/framework-detection.test.js

```diff
@@ -283,3 +285,21 @@ test('should filter frameworks with no dev command', async (t) => {
     })
   })
 })
+
+test('should pass framework-info env to framework sub process', async (t) => {
+  await withSiteBuilder('site-with-gatsby', async (builder) => {
+    await builder
+      .withPackageJson({
+        packageJson: {
+          dependencies: { gatsby: '^3.0.0' },
+          scripts: { develop: platform === 'win32' ? 'echo %GATSBY_LOGGER%' : 'echo $GATSBY_LOGGER' },
```

Comment:
Would the following work? 

```suggestion
          scripts: { develop: 'node -p process.env.GATSBY_LOGGER' },
```

Reply:
Done in https://github.com/netlify/cli/pull/3194/commits/1d0cdb51271a1dd4590c1c17d080b588e085460b